megamek/src/megamek/common/TripodMech.java

```diff
@@ -1282,7 +1290,8 @@ && getCrew().getOptions().booleanOption(
                         case 11:
                             return new HitData(Mech.LOC_LARM, true);
                         case 12:
-                            if (getCrew().hasEdgeRemaining()
+                            if (game.getOptions().booleanOption(OptionsConstants.EDGE)
+                                && getCrew().hasEdgeRemaining()
                                 && getCrew().getOptions().booleanOption(
                                     "edge_when_headhit")) {
```

Comment:
We should probably take this opportunity to replace all of these with ```EDGE_WHEN_HEADHIT```

Reply:
Good call, done in 62ec6e8